Lib/fontbakery/codetesting.py

```diff
@@ -96,8 +97,6 @@ def __call__(self, values, condition_overrides={}):
         if isinstance(values, str):
             if values.endswith('README.md'):
                 values = {'readme_md': values}
-            elif values.endswith('.ufo'):
-                values = {'ufo': values}
```

Comment:
why did you remove the ability to run code-tests on UFOs?

Reply:
Did I remove such ability? Wasn't intentional. Getting the UFO tests to pass again wasn't straightforward. I'll take another look.

Reply:
No worries. Thanks.